libraries/chain/webassembly/runtimes/eos-vm-oc/code_cache.cpp

```diff
@@ -106,7 +107,7 @@ std::tuple<size_t, size_t> code_cache_async::consume_compile_thread_queue() {
 }
 
 
-const code_descriptor* const code_cache_async::get_descriptor_for_code(const digest_type& code_id, const uint8_t& vm_version, bool is_write_window, get_cd_failure& failure) {
+const code_descriptor* const code_cache_async::get_descriptor_for_code(const account_name& receiver, const digest_type& code_id, const uint8_t& vm_version, bool is_write_window, get_cd_failure& failure) {
```

Comment:
I think it's fine to leave as is, but for increased abstraction maybe this just takes ` bool high_priority` instead, and the `receiver.prefix() == chain::config::system_account_name` check is then left outside of any `eos-vm-oc` code

Reply:
I like that. Didn't feel right including `chain::config` here.